lib/FunctionIsolation/IsolateFunctions.cpp

```diff
@@ -699,8 +702,8 @@ void IsolateFunctionsImpl::handleAnyPCJumps(efa::OutlinedFunction &Outlined,
 }
 
 bool IF::runOnModule(Module &TheModule) {
-  if (not TheModule.getFunction("root")
-      or TheModule.getFunction("root")->isDeclaration())
+  if (not getRegisteredIRHelper("root", TheModule)
+      or getRegisteredIRHelper("root", TheModule)->isDeclaration())
     return false;
```

Comment:
`root` is not an helper :thinking:

Reply:
Sure, that would lead to users being able to use `root` in the model. But I guess that's not a big deal.

Reply:
We never emit a decompiled `root`.
Even then, we could ban it in a more ad-hoc way.
`root` is not a helper and seeing it treated as a helper is confusing.

Reply:
Done.